website/docs/components/form/primitives/partials/guidelines/guidelines.md

```diff
@@ -9,6 +9,46 @@
 
 - Use helper text to give the user extra details about the data you’re asking them to input, e.g., formatting requirements such as MM-DD-YYYY.
 
+### Form::CharacterCount
```

Comment:
the name "character count" should be added to the page's frontmatter list of aliases